### ISSUE TITLE
build instructions / docker-compose up failing

### ISSUE THREAD

**Issue by heath-heb** [flipstone/orville#78]
I'm guessing `docker-compose up` is the method for spinning up a local dev environment? If so, it's failing for me with the following:

```sh
$ docker-compose up
Building dev
Step 1/2 : FROM haskell:7.10.3
 ---> 78ab9c2522c6
Step 2/2 : RUN apt-get update &&    apt-get install -y --no-install-recommends libpq5 libpq-dev &&    apt-get clean
 ---> Running in 88c499da497c
Get:1 http://security.debian.org jessie/updates InRelease [44.9 kB]
Ign http://httpredir.debian.org jessie InRelease
Get:2 http://httpredir.debian.org jessie-updates InRelease [16.3 kB]
Get:3 http://ppa.launchpad.net trusty InRelease [15.4 kB]
E: The method driver /usr/lib/apt/methods/https could not be found.
ERROR: Service 'dev' failed to build: The command '/bin/sh -c apt-get update &&    apt-get install -y --no-install-recommends libpq5 libpq-dev &&    apt-get clean' returned a non-zero code: 100
```

**Comment by qxjit** [comment]
Thanks for the heads up, Heath. I'm not sure the "method driver" error is related to at the moment. Maybe the install commands to build the simply aren't up to date with the rest of the world...

Are you experiencing this issue on the `master` branch, `0.9_development`, both?

**Comment by qxjit** [comment]
@heath-heb Yes, you probably do want to be working on `0.9_development`.  I managed to track down the problem and put a workaround in. One of the apt sources (FPCO) was causing the problem. It's an http source, but I'm guessing FP Complete put a redirect to HTTPS in place. Unfortunately, the base haskell image we're using doesn't have https support installed for apt.

Since we expect the `0.9_development` branch to replace `master` anyhow I just went ahead and removed the offending apt source so that the master branch can work again.
